franklin/builder/serializers.py

```diff
@@ -27,6 +27,18 @@ class Meta:
         }
 
 
+class EnvironmentStatusSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Environment
+        fields = ('status',)
+        extra_kwargs = {
+            "github_id": {
+                "validators": [],
+            },
+        }
```

Comment:
extra_kwargs not needed here. We did it above due to a bug. What this is trying to do is look for `github_id` on the model and tell the serializer to skip validation for it. You don't have that on this model.